study-builder/README.md

```diff
@@ -30,7 +30,11 @@ To deploy the [`Study builder`](/study-builder/) manually:
 1. [Create](https://cloud.google.com/compute/docs/instances/create-start-instance) a Compute Engine VM instance with a [static IP](https://cloud.google.com/compute/docs/ip-addresses/reserve-static-internal-ip-address) and read/write [access scopes](https://cloud.google.com/compute/docs/access/service-accounts#accesscopesiam) for Cloud Storage (make sure your VM’s [GCE service account](https://cloud.google.com/compute/docs/access/service-accounts#default_service_account) has the [`Storage Object Admin`](https://cloud.google.com/storage/docs/access-control/iam-roles) role for the bucket you created during `Study datastore` deployment)
 1. Verify that your VM instance has the `Stackdriver Logging API` write [access scope](https://cloud.google.com/compute/docs/access/service-accounts#accesscopesiam) (on by default) and that your VM’s [service account](https://cloud.google.com/compute/docs/access/service-accounts#default_service_account) has the [`Logs Writer`](https://cloud.google.com/logging/docs/access-control) role (off by default)
 1. Check out the latest code from the [FDA MyStudies repository](https://github.com/GoogleCloudPlatform/fda-mystudies/)
-1. Update user-facing text as desired, for example in [`termsAndCondition.jsp`](fdahpStudyDesigner/src/main/webapp/WEB-INF/view/termsAndCondition.jsp) and [`privacypolicy.jsp`](fdahpStudyDesigner/src/main/webapp/WEB-INF/view/privacypolicy.jsp)
+1. Update user-facing text as desired, for example in [termsAndCondition.jsp](fdahpStudyDesigner/src/main/webapp/WEB-INF/view/termsAndCondition.jsp), [copyright.jsp](fdahpStudyDesigner/src/main/webapp/WEB-INF/templates/copyright.jsp) and [privacypolicy.jsp](fdahpStudyDesigner/src/main/webapp/WEB-INF/view/privacypolicy.jsp)
```

Comment:
- Indent all these new bullets as sub-bullets under a new bullet that says "1. Optional. Customize branding by updating text and images"
- Make sure you use `` for all the file names
- Remove the organization name instructions about application.properties since that should be covered in the comments of that file
- Remove the words "ensure you" - for example the parenthetical would then be "keep the same file name"
- Is resolution of the images important? If so, please provide that information here

Reply:
Fixed review comment.
I guess, resolution is not important, as the logo sizes are vary.

Reply:
Only a few of these requests are addressed, @SR-btc to help

Reply:
****
![image](https://user-images.githubusercontent.com/65340564/108193531-9020a600-713b-11eb-939f-39a7ab200817.png)

- Indent all these new bullets as sub-bullets under a new bullet that says "1. Optional. Customize branding by updating text and images"
- Make sure you use `` for all the file names
- Remove the organization name instructions about application.properties since that should be covered in the comments of that file
- Remove the words "ensure you" - for example the parenthetical would then be "keep the same file name"

The above 4 points are addressed in StudyBuilder [Readme.md](https://github.com/GoogleCloudPlatform/fda-mystudies/blob/2.0.2-release-candidate/study-builder/README.md)